### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by khiemdavidnguyen**

### CODE REVIEW

File: Khiem_Nguyen/Python_3/Python/python_stack/python_fundamentals/for_loop_basic2.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+def analyze(arr):
+    myDict = {"sumTotal":"", "average":"", "minimum":"", "maximum":"", "length":""}
+    sum = 0
+    for i in range(0,len(arr)):
+        sum = sum + arr[i]
+        myDict["sumTotal"] = sum
+        average = sum/len(arr)
+        myDict["average"] = average
+        currentMin = arr[0]
+        for i in range(0,len(arr)):
```
Comment (ranriy) [inline Khiem_Nguyen/Python_3/Python/python_stack/python_fundamentals/for_loop_basic2.py:79]: You need not use nested loops. It's expensive. Draw out a T diagram for this code and refactor it!

File: Khiem_Nguyen/Python_3/Python/python_stack/python_fundamentals/for_loop_basic2.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        myDict["length"] = len(arr)
+    print(myDict)
+analyze([1,2,3,4])
+
+#9
+def reverse(arr):
+    if len(arr) % 2 == 0:
+        half = int(len(arr)/2) + 1
+    else:
+        half = int(len(arr)/2) + .5
```
Comment (ranriy) [inline Khiem_Nguyen/Python_3/Python/python_stack/python_fundamentals/for_loop_basic2.py:97]: + 0.5 is making it a float. Do we need + 0.5?

File: Khiem_Nguyen/Python_3/Python/python_stack/python_fundamentals/for_loop_basic2.py (hunk 1 above)
Comment (khiemdavidnguyen) [reply Khiem_Nguyen/Python_3/Python/python_stack/python_fundamentals/for_loop_basic2.py:79]: I removed the two "for" loops nested. I ran it and it's good now. I will commit again with changes. Thanks for feedback!

File: Khiem_Nguyen/Python_3/Python/python_stack/python_fundamentals/for_loop_basic2.py (hunk 2 above)
Comment (khiemdavidnguyen) [reply Khiem_Nguyen/Python_3/Python/python_stack/python_fundamentals/for_loop_basic2.py:97]: We don't need it. I just realized that "int()" will round the number to the nearest whole number? I  originally thought the "+ .5" was needed to remove the floating. But "int()" will make it an integer even if it's a decimal number. I ran it and it's good. I will recommit. Thanks for feedback!

**Pull request merged**